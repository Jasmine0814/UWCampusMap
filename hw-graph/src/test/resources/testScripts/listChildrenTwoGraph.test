# test the children of many nodes separate in two graphs can be listed.

CreateGraph A
CreateGraph B
AddNode A n1
AddNode A n2
AddNode A n3
AddNode B n4
AddNode B n5
AddNode B n6
AddEdge A n1 n2 e12
AddEdge A n1 n3 e13
AddEdge B n4 n5 e45
AddEdge B n4 n6 e16
ListChildren A n1
ListChildren B n4
