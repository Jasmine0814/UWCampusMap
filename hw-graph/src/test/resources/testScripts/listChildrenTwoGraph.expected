# test the children of many nodes separate in two graphs can be listed.

CreateGraph A
CreateGraph B
added node n1 to A
added node n2 to A
added node n3 to A
added node n4 to B
added node n5 to B
added node n6 to B
added edge e12 from n1 to n2 in A
added edge e13 from n1 to n3 in A
added edge e45 from n4 to n5 in B
added edge e46 from n4 to n6 in B
the children of n1 in A are: n2(e12) n3(e13)
the children of n4 in B are: n5(e45) n6(e46)

